nltk/compat.py

```diff
@@ -474,36 +483,43 @@ def iterkeys(d):
     """Return an iterator over the keys of a dictionary."""
     return getattr(d, _iterkeys)()
 
+
 def itervalues(d):
     """Return an iterator over the values of a dictionary."""
     return getattr(d, _itervalues)()
 
+
 def iteritems(d):
     """Return an iterator over the (key, value) pairs of a dictionary."""
     return getattr(d, _iteritems)()
 
 try:
     from functools import total_ordering
-except ImportError: # python 2.6
+except ImportError:  # python 2.6
     def total_ordering(cls):
         """Class decorator that fills in missing ordering methods"""
         convert = {
             '__lt__': [('__gt__', lambda self, other: not (self < other or self == other)),
-                       ('__le__', lambda self, other: self < other or self == other),
+                       ('__le__', lambda self, other: self <
+                        other or self == other),
                        ('__ge__', lambda self, other: not self < other)],
             '__le__': [('__ge__', lambda self, other: not self <= other or self == other),
-                       ('__lt__', lambda self, other: self <= other and not self == other),
+                       ('__lt__', lambda self, other: self <=
+                        other and not self == other),
                        ('__gt__', lambda self, other: not self <= other)],
             '__gt__': [('__lt__', lambda self, other: not (self > other or self == other)),
-                       ('__ge__', lambda self, other: self > other or self == other),
+                       ('__ge__', lambda self, other: self >
+                        other or self == other),
                        ('__le__', lambda self, other: not self > other)],
             '__ge__': [('__le__', lambda self, other: (not self >= other) or self == other),
-                       ('__gt__', lambda self, other: self >= other and not self == other),
+                       ('__gt__', lambda self, other: self >=
+                        other and not self == other),
                        ('__lt__', lambda self, other: not self >= other)]
```

Comment:
total_ordering decorator was a verbatim copy from stdlib: https://hg.python.org/cpython/file/2.7/Lib/functools.py#l53 - I think we shouldn't change indentation